README.md

```diff
@@ -19,35 +27,45 @@ git clone git@github.com:ops-guru/docker-hive.git
 Create image
 
 ```
-docker build . -t amzn-hive-image
+docker build . -t opsguruhub/docker-hive
 ```
 
+You may want to do some cleanup first:
+docker container stop local-hive; docker rm local-hive; docker rmi opsguruhub/docker-hive
+
 ## DockerHub
 
 Image available on DockerHub
 
 ```
-docker run -d opsguruhub/docker-hive:0.0.1
+docker pull opsguruhub/docker-hive:0.0.2
 ```
 
 ## Test
 
+Enusre that you don't have running container
+
+docker container stop local-hive; docker rm local-hive; 
+
 Run image
 
 ```
-CONTAINER_ID=$(docker run -d opsguruhub/docker-hive)
+docker run -p 8030-8088:8030-8088 -p 10000:10000 -p 10002:10002 -d --name local-hive opsguruhub/docker-hive
 ```
 
 Wait for services started
 
 ```
-docker logs $CONTAINER_ID
+docker logs local-hive
 ```
 
+Or you can open bash:
+docker exec -it local-hive bash
```

Comment:
@alex-ber , pls enclose it in backticks.

Reply:
Done